controllers/storagecluster/cephcluster.go

```diff
@@ -385,6 +381,23 @@ func validateMultusSelectors(selectors map[string]string) error {
 	return nil
 }
 
+// getNetworkSpec returns cephv1.NetworkSpec after reconciling the
+// storageCluster.Spec.HostNetwork and storageCluster.Spec.Network fields
+func getNetworkSpec(sc ocsv1.StorageCluster) cephv1.NetworkSpec {
+	// only hostnetwork specified
+	networkSpec := cephv1.NetworkSpec{HostNetwork: sc.Spec.HostNetwork}
```

Comment:
I'd complain about this override... but I suppose we're stuck with this field forever or until v2, which ever comes first. :stuck_out_tongue: